src/Components/ArticlePage/Suggested.jsx

```diff
@@ -0,0 +1,8 @@
+export const Suggested = () => {
+    return (
+        <div className="suggested">
+            <h2>Suggested Articles</h2>
+            <p>article placeholder</p>
+        </div>
+    )
+}
```

Comment:
You should only be pushing code that is relevant to the ticket you are working on, which this is not and should therefore not be included here